lib/pliny/metrics.rb

```diff
@@ -1,29 +1,48 @@
 module Pliny
   module Metrics
-    def self.count(*names, value: 1)
-      counts = Hash[names.map { |n| [metric_prefix(:count, n), value] }]
-      Pliny.log(counts)
+    extend self
+
+    attr_accessor :backends
+
+    @backends = []
+
+    def reset_backends
```

Comment:
I'm assuming this is only a handy helper for tests? If so, can this be a helper method in the specs instead?